### ISSUE TITLE
[QA] HLS support

### ISSUE THREAD

**Issue by sgregori** [AlexPresso/VideoStation-FFMPEG-Patcher#20]
**Question**

There are any patch for hls playing?

With 2.3.2 until 2.3.8 is working but not at newer versions.

**Comment by AlexPresso** [comment]
Good Evening,

I'm not sure to understand your question, the VideoStation's web player is already working over HLS (m3u8), the patcher is not changing the default VideoStation's behavior, only enabling some codecs and replacing default ffmpeg

**Comment by AlexPresso** [comment]
Are you using the default patching option or the wrapper one ?

**Comment by sgregori** [comment]
at this time, i just learning about your patch, because i surprised that the patch of libsynovte.so is done with "sed" on a binary file.

**Comment by sgregori** [comment]
```
# Patch libsynovte.so to authorize DTS, EAC3 and TrueHD
sed -i -e 's/eac3/3cae/' -e 's/dts/std/' -e 's/truehd/dheurt/' /var/packages/VideoStation/target/lib/libsynovte.so
```

**Comment by AlexPresso** [comment]
> but playing isnt available after 2.3.8 version.

Does it stop with an error message or is it loading forever ?

> when i use the libsynovte.so (2.3.8) with the all rest of package 2.4.0 or higher, playing is working but web interface crashes 

Do you mean the DS Video app (Android) is able to play the files, but the web versions crashes ? 

I'm not using the Android app at all, I'm expecting it to have the same behavior as the web version (HLS / .ts files with m3u8 playlist format) but they may have implemented another way to do it. That issue may also be related to the implementation of Advanced Media Extensions from Synology, which is the tool I'm still struggling with at the moment ( #16 )

I'll have a closer look at the differences between these two versions
